### PR TITLE
Add missing files to the imodel-bridge barrel.  Various other clean up

### PR DESCRIPTION
**Pull request by calebmshafer**
After reviewing the current Connector sample, found several missing exports from the barrel file and a couple re-exports of the same types.

Updated a few release tags while adding the files to the barrel.

@c-macdonald I didn't touch it but is there still a need for the `Main.ts` file?  Seems to be me it wouldn't technically be needed here.

### CODE REVIEW

File: core/imodel-bridge/src/BridgeRunner.ts [hunk 1]
```diff
@@ -22,22 +23,19 @@ import { IModelBridge } from "./IModelBridge";
...
+  /** Should be run after all documents have been synchronized.  Runs any actions (like project extent calculations) that need to run on the completed iModel */
   public allDocsProcessed: boolean = false;
-  /** Indicates whether the BridgeRunner should update the profile of the imodel's db. This would only need to be set to false if the imodel needs to be opened by legacy products */
+  /** Indicates whether the BridgeRunner should update the profile of the iModel's db. This would only need to be set to false if the iModel needs to be opened by legacy products */
   public updateDbProfile: boolean = true;
-  /** Indicates whether the BridgeRunner should update any of the core domain schemas in the imodel */
+  /** Indicates whether the BridgeRunner should update any of the core domain schemas in the iModel */
   public updateDomainSchemas: boolean = true;
-  /** The module containing the IModel Bridge implementation */
+  /** The module containing the iModel Bridge implementation */
```
Comment (c-macdonald) [inline core/imodel-bridge/src/BridgeRunner.ts:38]: 'Bridge' was changed to 'Connector' in other comments, but not in all of them.

**Comment by calebmshafer** [comment]
@akshay-madhukar @admccarthy1 FYI, I started this PR a while ago. Let me know if you want to chat about it or revive this.

Are you guys planning on addressing the question that Carole had some point in the future?

**Comment by calebmshafer** [comment]
@akshay-madhukar @jffmarker @zachzhu2016 I heard that there is going to be some work done related what I have proposed here.  Do you want me to keep my PR around so you can grab things from it or should I go ahead and abandon these changes?

**Comment by jffmarker** [comment]
This all seems reasonable, and @zachzhu2016 should take these changes before he does the rename I would think.

**Comment by zachzhu2016** [comment]
@jffmarker The changes made by Caleb are already in my PR. https://github.com/imodeljs/imodeljs/pull/1661

**Pull request closed without merging**